reference/api/faceting.md

```diff
@@ -0,0 +1,94 @@
+# Faceting
+
+_Child route of the [settings route](/reference/api/settings.md)._
+
+This route allows you to configure the faceting settings for an index.
+
+Faceting settings can also be updated directly through the [global settings route](/reference/api/settings.md#update-settings) along with the other settings.
+
+To learn more about filtering and faceting, refer to our [dedicated guide](/learn/advanced/filtering_and_faceted_search.md).
+
+::: warning
+Updating the settings means overwriting the default settings of Meilisearch. You can reset to default values using the `DELETE` routes.
+:::
+
+## Get faceting settings
+
+<RouteHighlighter method="GET" route="/indexes/{index_uid}/settings/faceting"/>
+
+Get the faceting settings of an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+
+### Example
+
+<CodeSamples id="get_faceting_settings_1" />
+
+#### Response: `200 OK`
+
+```json
+{
+  "maxValuesPerFacet": 100
+}
+```
+
+### Returned fields
+
+#### `maxValuesPerFacet`
+
+Maximum number of facet values returned for each facet.
+
+## Update faceting settings
+
+<RouteHighlighter method="PATCH" route="/indexes/{index_uid}/settings/faceting"/>
+
+Partially update the faceting settings for an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+
+### Body
+
+#### `maxValuesPerFacet`
+
+**Type:** integer
+**Default value:** `100`
```

Comment:
Should we mention that 100 is the recommended value? From the [spec (lines 31-33)](https://github.com/meilisearch/specifications/blob/fb8b067925afda0ec51855df87a165a50222da5b/text/157-faceting-setting-api.md#311-maxvaluesperfacet)

>The value of 100 ensures good performance and prevents malicious users from scraping data from a Meilisearch instance.
>Increasing this value can degrade performance as well as expose the data of an instance to scrapping.